### ISSUE TITLE
Research PyPI org repo transfers

### ISSUE THREAD

**Issue by tim-schilling** [django-commons/controls#3]

**Comment by tim-schilling** [comment]
I think this isn't a real feature yet as per the Django Chat podcast episode with Deb Nicholson.

**Comment by tim-schilling** [comment]
I got a more defined answer from a recent board member office hour today:

> The update on orgs AIUI is probably not going to be super satisfying, but essentially the process of setting up the ability to process payments was much more complex than anticipated, so rollout for paid users (i.e. companies) has been delayed. Having the new PyPI engineer role active has been an enormous help in chipping away at PyPI service backlogs to free up capacity for finishing this work--but there is no target date at this time.
TL;DR - Orgs rollout is coming, but no specific timeline available yet.

**Comment by tim-schilling** [comment]
@cunla I believe the closing action for this would be documenting the transfer for orgs in a playbook.

**Comment by cunla** [comment]
Ryan and I did a while ago.
https://github.com/django-commons/controls/commit/c5c13bc11f1c9ffeb511ae510dd88fa6306500cc

**Comment by tim-schilling** [comment]
Ah, sorry and thank you!
